examples/eidolon_examples/venture_search_agent/harmonic_api_logic_unit.py

```diff
@@ -0,0 +1,151 @@
+import os
+from typing import List, Optional, Any, Dict
+from urllib.parse import urljoin
+
+import jsonref
+from httpx import AsyncClient, Timeout
+from pydantic import BaseModel, Field
+
+from eidolon_ai_client.util.aiohttp import AgentError
+from eidolon_ai_client.util.logger import logger
+from eidolon_ai_sdk.cpu.call_context import CallContext
+from eidolon_ai_sdk.cpu.logic_unit import LogicUnit
+from eidolon_ai_sdk.system.fn_handler import FnHandler
+from eidolon_ai_sdk.system.reference_model import Specable
+from eidolon_ai_sdk.util.schema_to_model import schema_to_model
+
+
+class ApiLogicUnitSpec(BaseModel):
+    root_call_url: str
+    open_api_location: str
+    operations_to_expose: Dict[str, str]  # map of tool name to openapi path
```

Comment:
When we promote this into the actual SDK will will probably want to make this a pydantic object with two fields "tool_name" and "path" to make the yaml files less mysterious. It keeps in line with k8 preference of lists of objects with a name field rather than maps.